package/MDAnalysis/topology/TOPParser.py

```diff
@@ -148,6 +151,10 @@ class TOPParser(TopologyReaderBase):
     .. _`PARM parameter/topology file specification`:
        http://ambermd.org/formats.html#topology
 
+    Additionally, the %RESIDUE_CHAINID non-standard flag is supported. This
+    can be added with the addPDB command from parmed:
+    https://parmed.github.io/ParmEd/html/parmed.html#addpdb
```

Comment:
Please use named links instead of bare links (see example above with 
```reST
`PARM parameter/topology file specification`_
```

State what happens when %RESIDUE_CHAINID is not present.

Reply:
Done.